### ISSUE TITLE
nanoc view does not use default values

### ISSUE THREAD

**Issue by DivineDominion** [nanoc/nanoc#1441]
### Steps to reproduce

Run `nanoc view` without options

### Expected behavior

I expected it to use the default values and produce similar output to this call:

```
$ nanoc view -o 127.0.0.1 -p 3000
View the site at http://127.0.0.1:3000/
[2019-06-26 13:56:43] INFO  WEBrick 1.4.2
[2019-06-26 13:56:43] INFO  ruby 2.6.2 (2019-03-13) [x86_64-darwin18]
[2019-06-26 13:56:43] INFO  WEBrick::HTTPServer#start: pid=36366 port=3000
```

### Actual behavior

It crashes:

```
$ nanoc view

Captain! We’ve been hit!

KeyError: key not found: :host
```

### Details

Just updated from nanoc 4.11.2 to 4.11.5.

### Crash log

[crash.log](https://github.com/nanoc/nanoc/files/3329854/crash.log)

**Comment by ddfreyne** [comment]
This is a bug which I've fixed already (https://github.com/nanoc/nanoc/commit/ce9e8eedcfe8763ff2228a0d628afbcf3ab2b90a) but haven't been able to release yet. I am waiting for my MacBook to come back from repairs.

**Comment by ddfreyne** [comment]
A fix for this is released in Nanoc 4.11.6.
